sources/advanced/Command.ts

```diff
@@ -326,6 +326,15 @@ export abstract class Command<Context extends BaseContext = BaseContext> {
      */
     abstract async execute(): Promise<number | void>;
 
+    /**
+     * Standard error handler which will simply rethrow the error. Can be used to add custom logic to handle errors
+     * from the command or simply return the parent class error handling.
+     * @param error
+     */
+    async catch(error: any): Promise<void> {
+        throw error;
```

Comment:
I think the ideal would be to make the catch [here](https://github.com/arcanis/clipanion/blob/3df85d491bd0266fca025a71695d8a857a3afbb4/sources/advanced/Cli.ts#L257) (there's already a catch, after all!), and to turn the default implementation of `catch` into:
```suggestion
        this.context.stdout.write(this.cli.error(error, {command: this}));
```